proof/crefine/RISCV64/SR_lemmas_C.thy

```diff
@@ -1268,22 +1226,22 @@ lemma setObject_eq:
   apply (rule conjI)
    apply (rule_tac x = obj in exI)
    apply simp
-  apply (erule ssubst [OF ob])
-  done
+  by metis
 
 lemma getObject_eq:
   fixes ko :: "'a :: pspace_storable"
   assumes x: "\<And>q n ko. loadObject p q n ko =
-                (loadObject_default p q n ko :: 'a kernel)"
+                (loadObject_default p q n ko :: ('a :: pspace_storable) kernel_r)"
```

Comment:
```suggestion
                (loadObject_default p q n ko :: 'a :: pspace_storable kernel_r)"
```

Reply:
I think this appears in a few places, so I can fix all of these up as you suggest

Reply:
Yes, please. Too many parentheses :-)